src/com/owncloud/android/lib/resources/shares/CreateRemoteShareOperation.java

```diff
@@ -34,27 +34,57 @@
 import com.owncloud.android.lib.common.operations.RemoteOperationResult;
 import com.owncloud.android.lib.common.utils.Log_OC;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Locale;
+
 /**
  * Creates a new share.  This allows sharing with a user or group or as a link.
  */
 public class CreateRemoteShareOperation extends RemoteOperation {
 
     private static final String TAG = CreateRemoteShareOperation.class.getSimpleName();
 
+    private static final String PARAM_NAME = "name";
+    private static final String PARAM_PASSWORD = "password";
+    private static final String PARAM_EXPIRATION_DATE = "expireDate";
```

Comment:
Better the other way around, I'll change code reference from EXPIRATION to EXIRE. 'expireDate' is the name determined by the OCS Share API for the header, we need to use exactly that.